### ISSUE TITLE
`sort-imports` rule bugs out when using single and multiple values.

### ISSUE THREAD

**Issue by marbuser** [eslint/eslint#11159]
**Tell us about your environment**

* **ESLint Version:** 5.9.0
* **Node Version:** 11.3.0
* **npm Version:** 6.4.1 (I use Yarn which is v 1.12.3)

**What parser (default, Babel-ESLint, etc.) are you using?**
Default

**Please show your full configuration:**
<details>
<summary>Configuration</summary>

```js
module.exports = {
  root: true,
  env: {
    node: true,
  },
  extends: [
    'airbnb-base',
    'plugin:prettier/recommended'
  ],
  rules: {
    'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
    'prettier/prettier': [
      'error',
      {
        trailingComma: 'es5',
        jsxBracketSameLine: true,
        singleQuote: true,
        printWidth: 100,
      },
    ],
    "sort-imports": ["error", {
      "ignoreCase": true,
      "ignoreMemberSort": false,
      "memberSyntaxSortOrder": ["all", "single", "multiple", "none",]
  }]
  },
};
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
import Joi from 'joi';
import Sequelize from 'sequelize';
import { data } from '../data';
```

**What did you expect to happen?**
Nothing. The code given above is 'correct' according to the rules.

**What actually happened? Please include the actual, raw output from ESLint.**
ESLint complains that; 
```js
import { data } from '../data';
```
should be sorted alphabetically with the other imports even though in my config I've stated that it should be; 'all', 'single', 'multiple', 'none'.

**Are you willing to submit a pull request to fix this bug?**
I don't have any experience working development on ESLint or OSS in general, so no probably not.

**Comment by nzakas** [comment]
Thanks for the report. I'm not very familiar with this rule, but it looks like the rule wants you to reorder the code like this:

```js
import { data } from '../data';
import Joi from 'joi';
import Sequelize from 'sequelize';
```

I'm not sure if this is a bug or not, let's see if another team member who's more familiar with this rule can figure it out.
